src/main/java/net/pms/network/NetworkConfiguration.java

```diff
@@ -297,15 +298,24 @@ private void checkNetworkInterface(NetworkInterface networkInterface, String par
 		// Use networkInterface.getInetAddresses() instead
 		for (InetAddress address : Collections.list(networkInterface.getInetAddresses())) {
 			if (address != null) {
-				LOGGER.trace("checking {} on {}", new Object[] {address, networkInterface.getName()});
-
+				LOGGER.trace("checking {} on {}", address, networkInterface.getName());
 				if (isRelevantAddress(address)) {
 					// Avoid adding duplicates
 					if (!subAddress.contains(address)) {
 						LOGGER.trace("found {} -> {}", networkInterface.getName(), address.getHostAddress());
 						final InterfaceAssociation ia = new InterfaceAssociation(address, networkInterface, parentName);
 						interfaces.add(ia);
-						mainAddress.put(networkInterface.getDisplayName(), ia);
+						try {
+							// Add network interface to the list of interfaces with associated address only
+							// if interface is supporting multicast.
+							if (networkInterface.supportsMulticast()) {
+								interfacesWithAssociatedAddress.put(networkInterface.getDisplayName(), ia);
+								LOGGER.trace("added interface {} with associated address {} supporting multicast to the list of relevant addresses", networkInterface.getName(), ia.getAddr());
+							}
```

Comment:
```suggestion
							} else {
								LOGGER.trace("Ignoring interface {} with associated address {} because it does not seem to support multicast", networkInterface.getName(), ia.getAddr());
							}
```